docs/contributing/developing.md

```diff
@@ -44,7 +44,11 @@ Compile the assets:
 npm run build
 ```
 
-Any Wagtail sites you start up in this virtualenv will now run against this development instance of Wagtail. We recommend using the [Wagtail Bakery demo site](https://github.com/wagtail/bakerydemo/) as a basis for developing Wagtail. Keep in mind that the setup steps for a Wagtail site may include installing a release version of Wagtail, which will override the development version you've just set up. In this case, you should install the site before running the `pip install -e` step, or re-run that step after the site is installed.
+Any Wagtail sites you start up in this virtualenv will now run against this development instance of Wagtail. We recommend using the [Wagtail Bakery demo site](https://github.com/wagtail/bakerydemo/) as a basis for developing Wagtail. Keep in mind that the setup steps for a Wagtail site may include installing a release version of Wagtail, which will override the development version you've just set up. In this case, to install the local Wagtail development instance in your virtualenv for your Wagtail site:
+```sh
+pip install -e path/to/wagtail"[testing, docs]" -U
+```
+Here, `path/to/wagtail`is the path to your local Wagtail copy.
```

Comment:
```suggestion

Here, `path/to/wagtail`is the path to your local Wagtail copy.
```

Reply:
Nit: there's a missing space between `path/to/wagtail` and "is"

Reply:
I'll fix up a bit later - thanks Sage. Nice pickup

Reply:
Fixed in 9e016c17b857e87d3ff0ac5a242e6faaecfeb4a8 along with a few other white space issues.